### PR TITLE
print_qor: sorted tests by name and remove prefix

### PR DESCRIPTION
**Pull request by acomodi**
This PR improves the `print_qor` script in the following ways:

- sorts tests, so that they are more easily readable
- takes only the last two directories in the path (to avoid common paths to be added for each test
- collects only the relevant run-times in the P&R flow, excluding the overhead.

Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

### CODE REVIEW

File: utils/print_qor.py [hunk 1]
```diff
@@ -78,23 +77,32 @@ def main():
...
     w = csv.DictWriter(sys.stdout, fields)
     w.writeheader()
+    rows = list()
     for root, dirs, files in os.walk(args.build_dir):
         if 'pack.log' in files:
             d = {}
 
-            d['path'] = root
-            d['pack time (sec)'] = scan_runtime(os.path.join(root, 'pack.log'))
+            path = root.split('/')
```
Comment (litghost) [inline utils/print_qor.py:85]: Factor this into a function like `last_n_dirs(root, 2)`, etc.

File: utils/print_qor.py [hunk 2]
```diff
@@ -7,17 +7,16 @@
 import subprocess
 
 
-def scan_runtime(fname):
+def scan_runtime(step, fname):
```
Comment (litghost) [inline utils/print_qor.py:10]: Can you also output the overhead for each step.  So basically output step runtime and overhead runtime.  Make sense?

File: utils/print_qor.py [hunk 3]
```diff
@@ -78,23 +77,32 @@ def main():
...
+            )
             d['place time (sec)'] = scan_runtime(
-                os.path.join(root, 'place.log')
+                "Placement", os.path.join(root, 'place.log')
             )
             d['route time (sec)'] = scan_runtime(
-                os.path.join(root, 'route.log')
+                "Routing", os.path.join(root, 'route.log')
             )
             d['t_crit (ns)'], d['Fmax (MHz)'] = scan_critical(
```
Comment (litghost) [inline utils/print_qor.py:96]: Request in meeting to add geomean CPD as well as CPD to table.

File: utils/print_qor.py (hunk 1 above)
Comment (acomodi) [reply utils/print_qor.py]: Done

File: utils/print_qor.py (hunk 2 above)
Comment (acomodi) [reply utils/print_qor.py:10]: Sure, makes sense. Done

File: utils/print_qor.py (hunk 3 above)
Comment (acomodi) [reply utils/print_qor.py]: Added both CPD geomean and its corresponding frequency

**Review by litghost — COMMENTED** [review]
Looks good.  I want to review the output from CI, so I'll revisit this once the CI's return with results.

**Comment by acomodi** [comment]
@litghost All CI's went green:

for XC7, [here](https://storage.cloud.google.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/xc7/3073/20201117-112047/build/xc7_qor.csv) is the output of print_qor.

**Pull request merged**